content/post/2020-01-15-examples/index.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+slug: examples
+title: "Examples in R package"
```

Comment:
Maybe "Code examples in the R package manuals" or something similar is more specific.